charts/milvus/templates/config.tpl

```diff
@@ -75,6 +75,10 @@ pulsar:
 
 kafka:
   brokerList: {{ .Values.externalKafka.brokerList }}
+{{- if .Values.externalKafka.sasl.plain.username }}
+  saslUsername: {{ .Values.externalKafka.sasl.plain.username }}
```

Comment:
Add a `{{ end }}` here.

Reply:
my bad, first time playing around with helm charts :)

Thank you for catching this